README.md

```diff
@@ -42,21 +42,44 @@
         sudo apt-get update
         sudo apt-get install nginx
 
+* destination for a client is 127.0.0.1
+
+### Setup server using docker
+
+* Build serve part using docker
+
+        sudo docker build -t scrapy-bench-server .
+
+* Run docker container
+
+        sudo docker run --rm -ti scrapy-bench-server
+
+* Find out container's network address
```

Comment:
Quick question, can we continue to use ``localhost`` as the server address to keep things simpler? I think we don't have docker using localhost by default only on windows.